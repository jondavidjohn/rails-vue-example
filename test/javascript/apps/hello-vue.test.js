import { mount } from '@vue/test-utils'
import HelloVue from '@/apps/hello-vue'

describe('HelloVue', () => {
  it('should render', () => {
    const app = mount(HelloVue)
    expect(app).toMatchSnapshot()
  })
})
