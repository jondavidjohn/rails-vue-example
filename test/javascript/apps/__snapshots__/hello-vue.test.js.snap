// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloVue should render 1`] = `
<div>
  <h1>Hello Vue</h1>
</div>
`;
